internal/cmd/gocdk/testdata/biome.ct

```diff
@@ -19,21 +19,76 @@ $ cd myproj
 $ gocdk biome list
 dev
 
-$ gocdk biome add mybiome --launcher local
-gocdk: Adding biome "mybiome"...
+# "biome add" prompts for a biome name.
+$ echof stdin
```

Comment:
This is an example of creating a `stdin` file with a single `\n` in it, which simulates the user hitting enter once, accepting the default of `local`.